cpp/src/gandiva/compiled_expr.h

```diff
@@ -56,8 +96,20 @@ class CompiledExpr {
   // IR function in the generated code
   llvm::Function* ir_function_;
 
+  // IR function in the generated code which is for 16 bit selection vector
+  llvm::Function* ir_function_sel_vec_16_;
+
+  // IR function in the generated code whoch is for 32 bit selection  vector
```

Comment:
whoch -> which

Reply:
resolved it. Thank you for pointing it out.